dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_2_fat/test-applications/MpTelemetryLogApp/src/io/openliberty/microprofile/telemetry/logging/internal/fat/MpTelemetryLogApp/AccessServlet.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.logging.internal.fat.MpTelemetryLogApp;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
```

Comment:
Should all `javax` be changed to `jakarta`?

Reply:
The transformer will transform these into Jakarta, when the tests are repeated for every EE and MP versions.
This app is the same as the one found in `io.openliberty.microprofile.telemetry.2.0.logging.internal_fat`